src/browser/api/window.js

```diff
@@ -1187,7 +1187,7 @@ Window.moveBy = function(identity, deltaLeft, deltaTop) {
     let left = toSafeInt(deltaLeft, 0);
     let top = toSafeInt(deltaTop, 0);
 
-    browserWindow.setBounds({
+    browserWindow.setBounds({ // no need to call clipBounds as width and height not changing
```

Comment:
Sounds like an extra/unnecessary comment to me.
But if we  keep it, probably should go onto a separate line.